sqlglot/expressions.py

```diff
@@ -6221,11 +6221,11 @@ class Reduce(Func):
 class RegexpExtract(Func):
     arg_types = {
         "this": True,
-        "expression": True,
-        "position": False,
-        "occurrence": False,
-        "parameters": False,
-        "group": False,
+        "expression": True,  # The pattern
+        "position": False,  # Only start searching the string from this index
+        "occurrence": False,  # Skip the first `occurence-1` matches
+        "parameters": False,  # Flags, eg "i" for case-insensitive
```

Comment:
As a side note, would it be possible to rename this arguments? they seem to follow snowflake naming conventions, but I really find them pretty unclear. I would love to rename as

expression -> pattern
position -> start_position
occurrence -> occurrence
parameters -> flags
group -> group

Reply:
I'd personally leave them as they are, generally in AST nodes we try to be as concise as possible. In addition, that'd be a breaking change impacting users for no real reason.

Also a thought/nit, It's not often that we document each arg because we don't really use them ourselves, it tends to add noise.

Reply:
I'd personally leave them as they are, generally in AST nodes we try to be as concise as possible. In addition, that'd be a breaking change impacting users for no real reason.

Also a thought/nit, It's not often that we document each arg because we don't really use them ourselves, it tends to add noise.